chunk/chunk_store_test.go

```diff
@@ -146,3 +146,44 @@ func diff(want, have interface{}) string {
 	})
 	return "\n" + text
 }
+
+func TestBigBuckets(t *testing.T) {
```

Comment:
Really glad you wrote tests!

I found them a bit hard to understand their purpose at first, so I'm going to suggest some commentary inside.